### PR TITLE
OCPBUGS-23121: Restart operator pod on malformed RBAC

### PR DESCRIPTION
**Pull request by jrvaldes**
This PR triggers a restart after deleting malformed ClusterRoleBinding, so that the new instance of the operator pod reconciles with the newly created resources.

### CODE REVIEW

**Comment by jrvaldes** [comment]
/test aws-e2e-operator

to gather another data point

File: controllers/configmap_controller.go [hunk 1]
```diff
@@ -71,8 +72,8 @@ const (
 	UsernameAnnotation = "windowsmachineconfig.openshift.io/username"
 	// ConfigMapController is the name of this controller in logs and other outputs.
 	ConfigMapController = "configmap"
-	// wicdRBACResourceName is the name of the resources associated with WICD's RBAC permissions
-	wicdRBACResourceName = "windows-instance-config-daemon"
+	// WicdRBACResourceName is the name of the resources associated with WICD's RBAC permissions
+	WicdRBACResourceName = "windows-instance-config-daemon"
```
Comment (sebsoto) [inline controllers/configmap_controller.go:76]: Wicd -> WICD

File: hack/run-ci-e2e-test.sh [hunk 2]
```diff
@@ -110,6 +110,16 @@ if  ! oc get deploy/windows-machine-config-operator -n $WMCO_DEPLOY_NAMESPACE >
...
+
+  # OCPBUGS-23121: For AWS, test the operator installed in a different namespace after a successful deployment in the
+  # provided namespace
+  platform="$(oc get infrastructure cluster -ojsonpath={.spec.platformSpec.type})"
+  if [[ "$platform" = "AWS" ]]; then
+    cleanup_WMCO $OSDK
+    export WMCO_DEPLOY_NAMESPACE="${WMCO_DEPLOY_NAMESPACE}-customized"
+    run_WMCO $OSDK
+    get_WMCO_logs
+  fi
```
Comment (sebsoto) [inline hack/run-ci-e2e-test.sh:122]: Do we really need to add this test?
Our test suite is so complicated already and this is a true edge case

Comment (jrvaldes) [reply hack/run-ci-e2e-test.sh:122]: I had mixed feelings about the value of adding e2e coverage for this corner case. So, I proposed the test for one platform only (AWS).  

Happy to remove it from the PR if that is the consensus. @openshift/openshift-team-windows-containers please chime in.

Comment (mansikulkarni96) [reply hack/run-ci-e2e-test.sh:122]: I am fine with removing it.

Comment (jrvaldes) [reply hack/run-ci-e2e-test.sh]: Removed the test commits.
